src/appengine/handlers/cron/grouper.py

```diff
@@ -127,6 +128,11 @@ def matches_top_crash(testcase, top_crashes_by_project_and_platform):
 
 def _group_testcases_based_on_variants(testcase_map):
   """Group testcases that are associated based on variant analysis."""
+  # Skip this if the project is configured so (like Google3).
+  enable = local_config.ProjectConfig().get('deduplication.variant', True)
```

Comment:
FYI let's make this `deduplication.variant` too to make it a bit more structured.

Reply:
Noted! Updated the config accordingly.